### PR TITLE
[awscontainerinsights receiver]Add Readme for ECS

### PR DESCRIPTION
**Pull request by JohnWu20**
**Description:** <Describe what has changed.>

Add ReadMe for ECS EC2 instance of container insights receiver 

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by codeboten — APPROVED** [review]

File: receiver/awscontainerinsightreceiver/README.md [hunk 1]
```diff
@@ -659,4 +659,201 @@ kubectl apply -f config.yaml
...
+| instance_network_rx_packets             | Count/Second  |
+| instance_network_total_bytes            | Bytes/Second  |
+| instance_network_tx_bytes               | Bytes/Second  |
+| instance_network_tx_dropped             | Count/Second  |
+| instance_network_tx_errors              | Count/Second  |
+| instance_network_tx_packets             | Count/Second  |
+| instance_number_of_running_tasks        | Count         | 
+
+
+<br/><br/>
```
Comment (rakyll) [inline receiver/awscontainerinsightreceiver/README.md:755]: Remove break lines, these are unnecessary.

**Review by rakyll — COMMENTED** [review]
Thanks for the doc, I don't know enough to comment on the content but left a styling tip.

File: receiver/awscontainerinsightreceiver/README.md [hunk 2]
```diff
@@ -659,4 +659,201 @@ kubectl apply -f config.yaml
 | container_status_reason           |
 | container_last_termination_reason | 
 
-The attribute `container_status_reason` is present only when `container_status` is in "Waiting" or "Terminated" State. The attribute `container_last_termination_reason` is present only when `container_status` is in "Terminated" State.
\ No newline at end of file
+The attribute `container_status_reason` is present only when `container_status` is in "Waiting" or "Terminated" State. The attribute `container_last_termination_reason` is present only when `container_status` is in "Terminated" State.
+
+This is a sample configuration for AWS Container Insights using the `awscontainerinsightreceiver` and `awsemfexporter` for an ECS cluster:
```
[the remark this answers is not in the document]
Comment (JohnWu20) [reply receiver/awscontainerinsightreceiver/README.md:664]: Sure, will make it clear

File: receiver/awscontainerinsightreceiver/design.md [hunk 3]
```diff
@@ -38,3 +40,9 @@ The following two packages are used to decorate metrics:
     * add the corresponding attribute `Service` to the metrics for the relevant pods
     * call "List & Watch" for "Endpoint" objects from Kubernetes API server. In this mode, when there is any change to “Endpoint”, agent will get the notification of the change. The benefit is that the agent could get Endpoint information update in time. If the "Endpoint" resources just occasionally slightly changes, the impact to API server is supposed to be minimal.
 
+The following package is to collect AWS ECS specific metrics and AWS ECS relative resources:
```
[the remark this answers is not in the document]
Comment (JohnWu20) [reply receiver/awscontainerinsightreceiver/design.md:43]: Thanks will change it.

**Comment by JohnWu20** [comment]
Hi, can anyone help review this pr? It is marked as stale now. If there is no questions about it, can we get it merged?

**Review by codeboten — COMMENTED** [review]
The PR looks good to me, pinging @Aneurysm9 @pxaws as they are the codeowners for this receiver.

**Pull request merged**